docs/server/java/_getFeatureGate.mdx

```diff
@@ -0,0 +1,57 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# `APIFeatureGate` Data Class Overview
+
+The `APIFeatureGate` data class encapsulates the evaluation result of a feature gate for a specific user. It provides detailed information regarding whether the feature gate is enabled or disabled etc.
```

Comment:
The enabled/disabled is no different than `checkGate` - its mostly about getting metadata about the gate